controller/graphql/server.go

```diff
@@ -10,6 +11,9 @@ import (
 	"github.com/graphql-go/graphql"
 )
 
+//go:embed index.html
+var index embed.FS
```

Comment:
I guess it's easier to use a FS for this, even if it's just one file, so we can use http.FileServer?

Reply:
right. otherwise it's a custom handler wrapping sending the bytes, right?